### ISSUE TITLE
Ability to Reuse Fields from Steps in Others

### ISSUE THREAD

**Issue by kahboom** [ZimaraIO/zimara-ui#15]
ie new product listing -> create a banner with it or social media post

Example:
User creates a product on Shopify. News of product creation is automatically shared on Twitter and a Telegram channel with the message "Hey there, my design is now on a new product. Visit my {shop_name} to get this cool {product_name} and check out more {product_category}! {shop_link}". You can make the message even more appealing by adding a Bannerbear node that automatically creates template images for your new product announcements.

**Comment by Delawen** [comment]
@igarashitm this is part of https://github.com/KaotoIO/kaoto-ui/issues/253 right?

**Comment by igarashitm** [comment]
Yep at least it needs Data Mapper capability, or it could be beyond that? It sounds like a complete integration story rather than a single feature.
